core/src/main/java/org/apache/carbondata/core/metadata/schema/datamap/DataMapClassProvider.java

```diff
@@ -31,7 +31,8 @@
   TIMESERIES("org.apache.carbondata.core.datamap.TimeSeriesDataMap", "timeseries"),
   LUCENE("org.apache.carbondata.datamap.lucene.LuceneFineGrainDataMapFactory","lucene"),
   BLOOMFILTER("org.apache.carbondata.datamap.bloom.BloomCoarseGrainDataMapFactory", "bloomfilter"),
-  MV("org.apache.carbondata.core.datamap.MVDataMap", "mv");
+  MV("org.apache.carbondata.core.datamap.MVDataMap", "mv"),
+  MV_TIMESERIES("org.apache.carbondata.core.datamap.MVTimeSeriesDataMap", "mv_timeseries");
```

Comment:
@Indhumathi27 : MV is a generic concept. It can handle any queries  (even timeseries). Timeseries is just a UDF. so  MVDataMap itself should handle this instead of adding new MVTimeSeriesDataMap

Reply:
yes. removed mv_timeseries provider and added validation's for timeseries query in this PR